docs/README.md

```diff
@@ -1,64 +1,49 @@
 
-## Organisations
+### Organisations
 
-[![Primary](https://opencollective.com/ngxs/sponsor/0/avatar.svg)](https://opencollective.com/ngxs/sponsor/0/website)
-[![Sponsor 1](https://opencollective.com/ngxs/sponsor/1/avatar.svg)](https://opencollective.com/ngxs/sponsor/1/website)
-[![Sponsor 2](https://opencollective.com/ngxs/sponsor/2/avatar.svg)](https://opencollective.com/ngxs/sponsor/2/website)
-[![Sponsor 3](https://opencollective.com/ngxs/sponsor/3/avatar.svg)](https://opencollective.com/ngxs/sponsor/3/website)
-[![Sponsor 4](https://opencollective.com/ngxs/sponsor/4/avatar.svg)](https://opencollective.com/ngxs/sponsor/4/website)
-[![Sponsor 5](https://opencollective.com/ngxs/sponsor/5/avatar.svg)](https://opencollective.com/ngxs/sponsor/5/website)
-[![Sponsor 6](https://opencollective.com/ngxs/sponsor/6/avatar.svg)](https://opencollective.com/ngxs/sponsor/6/website)
-[![Sponsor 7](https://opencollective.com/ngxs/sponsor/7/avatar.svg)](https://opencollective.com/ngxs/sponsor/7/website)
-[![Sponsor 8](https://opencollective.com/ngxs/sponsor/8/avatar.svg)](https://opencollective.com/ngxs/sponsor/8/website)
-[![Sponsor 9](https://opencollective.com/ngxs/sponsor/9/avatar.svg)](https://opencollective.com/ngxs/sponsor/9/website)
+[![Primary](https://opencollective.com/ngxs/sponsor/0/avatar.svg)](https://opencollective.com/ngxs/sponsor/0/website) [![Sponsor 1](https://opencollective.com/ngxs/sponsor/1/avatar.svg)](https://opencollective.com/ngxs/sponsor/1/website) [![Sponsor 2](https://opencollective.com/ngxs/sponsor/2/avatar.svg)](https://opencollective.com/ngxs/sponsor/2/website) [![Sponsor 3](https://opencollective.com/ngxs/sponsor/3/avatar.svg)](https://opencollective.com/ngxs/sponsor/3/website) [![Sponsor 4](https://opencollective.com/ngxs/sponsor/4/avatar.svg)](https://opencollective.com/ngxs/sponsor/4/website) [![Sponsor 5](https://opencollective.com/ngxs/sponsor/5/avatar.svg)](https://opencollective.com/ngxs/sponsor/5/website) [![Sponsor 6](https://opencollective.com/ngxs/sponsor/6/avatar.svg)](https://opencollective.com/ngxs/sponsor/6/website) [![Sponsor 7](https://opencollective.com/ngxs/sponsor/7/avatar.svg)](https://opencollective.com/ngxs/sponsor/7/website) [![Sponsor 8](https://opencollective.com/ngxs/sponsor/8/avatar.svg)](https://opencollective.com/ngxs/sponsor/8/website) [![Sponsor 9](https://opencollective.com/ngxs/sponsor/9/avatar.svg)](https://opencollective.com/ngxs/sponsor/9/website)
 
-## Individuals
+### Individuals
 
 [![Backers](https://opencollective.com/ngxs/backers.svg?width=890)](https://opencollective.com/ngxs#backers)
 
-# Contributors
+## Contributors
 
 Thanks to all our [contributors](https://github.com/ngxs/store/graphs/contributors)! Open source does not work without you!
```

Comment:
Could we add the contributors with small pictures? this would add some visibility of our contributors

Reply:
I toggled the rich diff feature to see the size of the contributors' avatars. I am not sure how to make them smaller. I am afraid though, that a smaller avatar won't be sufficient. :/ 


![image](https://github.com/ngxs/store/assets/7045092/63eeb80e-6b90-453b-8daa-a1be6ca97d8f)